### PR TITLE
[test] Update dependencies

### PR DESCRIPTION
**Pull request by jribbink**
## Description

Update to latest Flow dependencies

______

- [ ] Targeted PR against `master` branch
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work
- [ ] Code follows the [standards mentioned here](https://github.com/onflow/cadence-lint/blob/master/CONTRIBUTING.md#styleguides)
- [ ] Updated relevant documentation 
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Added appropriate labels

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -6,38 +6,16 @@ on:
...
+      - 'test/v**'
   pull_request:
     branches:
       - master
       - 'feature/**'
       - 'v**'
+      - 'test/v**'
 
 jobs:
-  paths-filter:
```
Comment (turbolent) [inline .github/workflows/ci.yml:16]: What is the reasoning behind removing the paths filter?

**Review by turbolent — COMMENTED** [review]
Version bump looks good!

File: .github/workflows/ci.yml (hunk 1 above)
Comment (jribbink) [reply .github/workflows/ci.yml:16]: no reason, it must have inadvertantly got removed 🙃

Comment (turbolent) [reply .github/workflows/ci.yml:16]: Sounds good, no worries!

**Review by turbolent — APPROVED** [review]

**Pull request merged**